testsuite/tests/typing-unique/unique_analysis.ml

```diff
@@ -422,7 +422,7 @@ let foo () =
 Line 6, characters 12-13:
 6 |   unique_id r
                 ^
-Error: The value is used as unique, but part of it has already been used here:
+Error: The value is used as unique, but it has already been accessed here:
```

Comment:
This one is still confusing without the mention of a closure or function, because usually it is fine to access something before it is used. "This value is used as unique, but it is accessed within a closure here:" would be better. I think that you can probably detect this based solely on it being a shared use of the memory address, since I think that can only happen via a closure. Alternatively you could track this more directly in the analysis.

Reply:
This is addressed in the latest commit.